src/sysext/sysext.c

```diff
@@ -1658,6 +1658,11 @@ static int merge_subprocess(
                 if (r < 0)
                         return log_oom();
 
+                /* Also get the absolute file name with version info for logging. */
+                r = strv_extend(&extensions_v, basename(img->path));
```

Comment:
please do not use basename() anymore in our tree. Use path_extract_filename().

We are trying to get rid of all basename() use actually, across the whole codebase, because it's really icky in cornercases